references/product-recommendations-v1/.github/workflows/devrel-static-checks.yml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 Google LLC
```

Comment:
No need for this file. We run the linter on the repo level

Reply:
OK I'll remove.